src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = [];
+  const splitDate = date.split(fromFormat.pop());
+  const dateFormat = toFormat.pop();
+
+  let year,
+    month,
+    day;
```

Comment:
Better to declare variables separately.
```suggestion
  let year;
  let month;
  let day;
```